src/components/modules/auth/ui/pages/ResetPasswordClient.tsx

```
@@ -40,17 +35,17 @@ export function ResetPasswordClient() {
-            Please don't recharge the page
-          </p>
-        </CardContent>
-      </Card>
-    </div>
+        <Card className="w-full max-w-md text-center">
+          <CardHeader>
+            <CardTitle className="text-2xl">Validating link...</CardTitle>
+          </CardHeader>
+          <CardContent className="flex flex-col items-center gap-6">
+            <p className="text-muted-foreground">
+              Please don't recharge the page
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)